lib/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2020",
-    "module": "es2020",
+    "module": "CommonJS",
```

Comment:
Nice work spotting this 👍

Reply:
This change seems to introduce a large amount of compilation errors on the demo dapp 

![image](https://user-images.githubusercontent.com/3913019/184542336-d921f4a7-8932-4da6-8a4a-fa8029f0a314.png)

Reply:
I'll check this out. There is a way to generate both CommonJS and ESM modules in the same package.

Reply:
This is fixed now. Since there are two module standards, the normal import will import an ESM module to preserve backwards compatibility. Those who need the CommonJS module can import it using a deep import; `import { HashConnect } from 'hashconnect/dist/cjs/main';`